deploy/docker/fs/opt/appsmith/generate-infra-details.sh

```diff
@@ -14,15 +14,21 @@ function get_cloud_provider() {
      cloud_provider="azure";
   elif [[ $release_details == *"cloud"* ]];then
      cloud_provider="gcp";
+  elif [[ $release_details == *"generic"* ]];then
+     cloud_provider="digitalocean"
+  elif [[ $release_details == *"ecs"* ]];then
+     cloud_provider="alibaba"
```

Comment:
This `if/elif` chain is quickly getting out of hand. Can we add a comment to these with a full example string to help a bit please? Like:

```suggestion
     cloud_provider="azure";
  elif [[ $release_details == *"cloud"* ]];then
    # Example: abcd.cloud-magic.com
     cloud_provider="gcp";
  elif [[ $release_details == *"generic"* ]];then
    # Example: generic-cant-be-bothered-to-name-better.com
     cloud_provider="digitalocean"
  elif [[ $release_details == *"ecs"* ]];then
     cloud_provider="alibaba"
```

Can we fix the indentation as well please?